packages/expo-constants/src/Constants.ts

```diff
@@ -106,11 +108,48 @@ const constants = {
     }
     return nativeConstants.linkingUri;
   },
+  get manifest(): AppManifest | null {
+    const maybeManifest = getManifest();
+    if (!maybeManifest || !isAppManifest(maybeManifest)) {
+      return null;
+    }
+    return maybeManifest;
+  },
+  get manifest2(): Manifest | null {
+    const maybeManifest = getManifest();
+    if (!maybeManifest || !isManifest(maybeManifest)) {
+      return null;
+    }
+    return maybeManifest;
+  },
```

Comment:
Docblocks on these two methods would help a lot since "manifest2" otherwise doesn't make sense to someone who's unaware of the change from classic updates to modern updates.